public/staticwebapp.config.json

```diff
@@ -1,9 +1,9 @@
 {
-  "navigationFallback": {
-    "rewrite": "/index.html",
-    "exclude": ["/*.{png,jpg,gif,ico}", "/static/*"]
-  },
-  "globalHeaders": {
-    "content-security-policy": "default-src https: 'unsafe-eval' 'unsafe-inline'; object-src 'none'"
-  }
-}
+    "navigationFallback": {
+        "rewrite": "/index.html",
+        "exclude": ["/*.{png,jpg,gif,ico}", "/static/*"]
+    },
+    "globalHeaders": {
+        "content-security-policy": "img-src 'self' 'unsafe-inline' data: https:; script-src *.twimg.com *.twitter.com *.google-analytics.com 'self' 'unsafe-inline'; style-src *.twimg.com *.twitter.com 'self' 'unsafe-inline'; default-src *.twitter.com 'self'; connect-src *.vaccinehunters.ca *.azurewebsites.net 'self';"
```

Comment:
Can we get rid of the `*azurewebsites.net` in the `connect-src` whitelist? Or constrain it to a subdomain? I can't recall why i added it, but anyone can make a website in azure and get a domain that satisfies that constraint.